packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -92,7 +98,7 @@ async function createOutgoingPayment(
   deps: ServiceDependencies,
   options: CreateOutgoingPaymentOptions
 ): Promise<OutgoingPayment | OutgoingPaymentError> {
-  const grantId = options.grant ? options.grant.grant : undefined
+  const grantId = options.grant?.id
```

Comment:
Will this be required in the future? i.e. will this end up throwing if there is no grant on the ctx?

Reply:
I think `grant` is mainly optional for creating outgoing payments via the admin api
https://github.com/interledger/rafiki/blob/e773fb7cd533a1a8495e056987a772154651130b/packages/backend/src/graphql/schema.graphql#L97-L99